spec/core/1.0.md

```diff
@@ -405,6 +407,10 @@ Each pair of messages is to be considered individually.
 --> "have" is invalid; "have" argument "qux1.0" does not correspond to any "want" argument
 ```
 
+The server is under no obligation of agreeing to any module or capability.
+Clients should be prepared to gracefully handle unavailable modules or capabilities.
+If the client finds that the server has not agreed to a module or capability that the client needs, it must exit.
```

Comment:
Didn't you mean MUST instead of must?

Reply:
No. The uppercase words are for when rigor is required on the implementor's side, or when it is unclear whether something is optional or required.

If the client is at the point where they absolutely need some module/cap and the server does not agree to it, they have no chance but to die anyway, so we need not specify it anymore. The paragraph just clarifies to the reader that this is, in fact, the intended behavior, and not an oversight on the part of the standards editors.

Reply:
I've merged this now. If you disagree with my answer, we can continue the discussion regardless of that.

Reply:
Nah, dude. All good.